src/content/docs/en/pages/devtools/cli/azion-cli-alpha/cache-settings/cache-settings.mdx

```diff
@@ -0,0 +1,302 @@
+
+#### help
+
+The `--help` option displays more information about the `azion list cache-setting` command.
+
+#### order-by
+
+The `--order-by` option sorts the output based on the selected field.
+
+#### page
+
+The `--page` option returns a page of the list according to its number. The default value is 1.
+
+#### page-size
+
+The `--page-size` option defines how many items should be returned per page. The default value is 10.
+
+#### sort
+
+The `--sort` option defines the order of the items on the list. The value should be either `asc` or `desc`.
+
+---
+
+## Describe
+
+### Usage
+
+```bash
+$ azion describe cache-setting --application-id 1673635839 --cache-setting-id 107313
+```
+
+### Required flags
+
+#### application-id
+
+The `--application-id` flag sets the unique identifier for the edge application.
+
+#### cache-setting-id
+
+The `--cache-setting-id` flag sets the unique identifier for a Cache Settings configuration.
+
+### Optional flags
+
+#### format
+
+The `--format` option changes the output format. Pass the value `json` to the flag to get the output in JSON format.
+
+#### help
+
+The `--help` option displays more information about the `azion describe cache-setting` command.
+
+#### out
+
+The `--out` option exports the output to the given file path or file name with extension. For example, `--out "./tmp/test.json"`.
+
+---
+
+## Update
+
+### Usage
+
+```bash
+$ azion update cache-setting --application-id 1673635839 --cache-setting-id 123123421 --name "phototypesetting"
+```
+
+### Required flags
+
+#### application-id
+
+The `--application-id` flag sets the unique identifier for the edge application.
+
+#### cache-setting-id
+
+The `--cache-setting-id` flag sets the unique identifier for a Cache Settings configuration.
+
+#### name
+
+The `--name` flag sets the name of the Cache Settings configuration being updated.
+
+### Optional flags
+
+#### adaptive-delivery-action
+
+The `--adaptive-delivery-action` option informs the Cache Settings configuration adaptive delivery action. The default value is `ignore`.
+
+#### browser-cache-settings
+
+The `--browser-cache-settings` option configures the amount of time that the content is cached in the web browser. The default value is `honor`.
+
+#### browser-cache-settings-maximum-ttl
+
+The `--browser-cache-settings-maximum-ttl` option informs Browser Cache Settings configuration maximum TTL.
+
+#### cache-by-cookies
+
+The `--cache-by-cookies` option defines if cache by cookies is active. The default value is `ignore`.
+
+#### cache-by-query-string
+
+The `--cache-by-query-string` option defines how the content should be cached according to variations of the Query String in URLs. The default value is `ignore`.
+
+#### cdn-cache-settings
+
+The `--cdn-cache-settings` option configures the amount of time Azion's Edge Applications take to cache the content. It can either Honor Origin Cache Headers or Override Cache Settings. The default value is `honor`.
```

Comment:
```suggestion
The `--cdn-cache-settings` option configures the amount of time Azion's edge applications take to cache the content. It can either Honor Origin Cache Headers or Override Cache Settings. The default value is `honor`.
```